src/core/ext/client_config/channel_connectivity.c

```diff
@@ -189,9 +189,9 @@ void grpc_channel_watch_connectivity_state(
   GRPC_API_TRACE(
       "grpc_channel_watch_connectivity_state("
       "channel=%p, last_observed_state=%d, "
-      "deadline=gpr_timespec { tv_sec: %lld, tv_nsec: %d, clock_type: %d }, "
+      "deadline=gpr_timespec { tv_sec: %"PRId64", tv_nsec: %d, clock_type: %d }, "
       "cq=%p, tag=%p)",
-      7, (channel, (int)last_observed_state, (long long)deadline.tv_sec,
+      7, (channel, (int)last_observed_state, deadline.tv_sec,
           (int)deadline.tv_nsec, (int)deadline.clock_type, cq, tag));
```

Comment:
for tv_nsec, do you need the cast?

It looks like that in other places it works without (int) cast. 
https://github.com/grpc/grpc/search?utf8=%E2%9C%93&q=PRId64